test/fetch-test-deps.sh

```diff
@@ -1,10 +1,50 @@
 #!/usr/bin/env bash
-
 set -e
 
 cd "$(dirname "$0")"
 
-case "$1" in
+# Parse arguments and provide usage message #########################
+
+# Provide a usage message
```

Comment:
```suggestion
```

It's an 80-line script, it doesn't need large section-heading comments. And the name `usage` followed by a few `echo`s is self-explanatory.